opencog/rule-engine/UREConfigReader.cc

```diff
@@ -79,10 +83,10 @@ void UREConfigReader::set_maximum_iterations(int mi)
 HandleSeq UREConfigReader::fetch_rules(Handle rbs)
 {
 	// Retrieve rules
-	Handle rule_var = _as.add_node(VARIABLE_NODE, "__URE_RULE__"),
-		rule_pat = _as.add_link(MEMBER_LINK, rule_var, rbs),
-		gl = _as.add_link(BIND_LINK, rule_pat, rule_pat),
-		rule_names = bindlink(&_as, gl);
```

Comment:
it was hard to read. When I saw the comma, I looked for the matching paren, and there were none... took a while to figure out why not.

Perhaps you meant to write this instead:
```
Handle rule_pat=_as.add_link(MEMBER_LINK,  _as.add_node(VARIABLE_NODE, "__URE_RULE__"), rbs);
Handle rule_names = bindlink(&_as, _as.add_link(BIND_LINK, rule_pat, rule_pat));
```
See what I mean about commas?  they are usually surrounded by parens.  Its weird when they are not.